### ISSUE TITLE
Latest build is not downloadable from https://firebase.tools/bin/linux/latest

### ISSUE THREAD

**Issue by lukeshan-ctm** [firebase/firebase-tools#7865]
### [REQUIRED] Environment info

**firebase-tools: 13.23.0**

**Platform: Linux**

### [REQUIRED] Steps to reproduce

Our build pipeline is trying to download from https://firebase.tools/bin/linux/latest

It redirect to https://github.com/firebase/firebase-tools/releases/download/v13.23.0/firebase-tools-linux which is not exist.

### [REQUIRED] Expected behavior

Firebase tools can be downloaded.

### [REQUIRED] Actual behavior

Firebase tools can not be downloaded.

**Comment by lukeshan-ctm** [comment]
This looks like the same issue as https://github.com/firebase/firebase-tools/issues/7648

**Comment by JoshStrobl** [comment]
Yep, CI / CD busted _again_. You'd expect they would sanity check after a new release, especially after the last time this was an issue.

**Comment by davidgilbertson** [comment]
Is this the same issue causing the Windows binary download link in the [Firebase docs](https://firebase.google.com/docs/cli#install-cli-windows) to fail?

**Comment by dcristolovean** [comment]
macos tools are missing from the last version, CI fails.  Shouldn't a new version be released with all the tools ?

**Comment by aalej** [comment]
Hey everyone, thanks for reporting this and sorry to hear that you encountered this issue. It looks like there might have been an issue with uploading the binaries for the latest version. I’ll reach out to our engineering team to upload the binaries to resolve this.

**Comment by UrbanovichA** [comment]
Same is for MacOS

**Comment by aalej** [comment]
Hey folks, thank you all for your patience. The binaries for the latest version have been uploaded. I’ve verified locally that `curl -sL https://firebase.tools | bash` now downloads the latest version of Firebase CLI. 

I’ll go ahead with this since this seems to be resolved. If you're still encountering an error, please let us know so that we can re-open this and take a look.

**Comment by aalej** [comment]
Thanks for the suggestion @vasilich6107. This type of issue is definitely not something we want our developers to keep experiencing, and we should find a way to mitigate this in the future. I’ll create a separate report so that we can keep track of this.
